murmur.go

```diff
@@ -38,11 +38,11 @@ func (d *digest) Write(p []byte) (n int, err error) {
 	if len(d.tail) > 0 {
 		// Stick back pending bytes.
 		nfree := d.Size() - len(d.tail) // nfree ∈ [1, d.Size()-1].
-		if nfree < len(p) {
+		if nfree < n {
 			// One full block can be formed.
 			block := append(d.tail, p[:nfree]...)
 			p = p[nfree:]
-			_ = d.bmix(block) // No tail.
```

Comment:
I'm not the code author, so it's only my guess why this assignment is used here. Assigning to `_` signifies that use a function with a non-void return and knowingly discard the return value.

Reply:
I didn't get you. It does not make any difference from the compiler's perspective, nor does it make the code more readable. Why would you want to show that you are knowingly discarding the return value when you are not using it in the first place ?

Reply:
For some it does make the code more readable – without reading `bmix` definition one can tell that it does return a value.

Reply:
It's been two years anyways since the PR was opened. Will just close this for now.